f3/granite.go

```diff
@@ -319,17 +320,14 @@ func (i *instance) isJustified(msg *GMessage) bool {
 		if msg.Round == 0 || msg.Value.IsZero() {
 			return false
 		}
-		prevRound := i.roundState(msg.Round - 1)
-		return prevRound.prepared.HasStrongQuorumAgreement(msg.Value.Head().CID) ||
-			prevRound.committed.HasStrongQuorumAgreement(ZeroTipSetID())
+		return true
```

Comment:
The comments on this method at 311 and some cases like 317 and 328 are now out of date. They're abstractly correct, but don't match the code so are confusing here. To be even more clear, how about:
- Move the validity tests (Round and Value.IsZero for some phases) to isValid
- Change this method to just `return true` (to be replaced by explicit justification later)

Reply:
Done!